statsmodels/sandbox/stats/multicomp.py

```diff
@@ -153,8 +153,13 @@ def get_tukeyQcrit2(k, df, alpha=0.05):
 
     not enough error checking for limitations
     '''
-    from statsmodels.stats.libqsturng import qsturng
-    return qsturng(1-alpha, k, df)
+    try:
+        # Studentized Range is not available below SciPy 1.7.
+        from scipy.stats import studentized_range
+        return studentized_range.ppf(1 - alpha, k, df)
+    except ImportError:
+        from statsmodels.stats.libqsturng import qsturng
+        return qsturng(1-alpha, k, df)
```

Comment:
Instead of this, how about at the top:
```python3
try:
    from scipy.stats import studentized_range
    qsturng = studentized_range.ppf
    psturng = studentized_rang.sf
except ImportError:
    from statsmodels.stats.libqsturng import qsturng, psturng
```
Then just call `qsturng`/`psturng` in the functions.
Note the use of `sf` instead of `1-cdf`. Same thing, but sometimes `sf` is implemented differently for better precision. Not currently true for `studentized_range`, but it could be someday.